ocean/helium/api.py

```diff
@@ -189,15 +284,66 @@ def _repr_html_(self):
 
 
 def snapshot(path, message):
+    """Creates a snapshot of a T4 object
+
+    Parameters:
+        path (str): path to snapshot
+        message (str): message to be associated with this snapshot
+    Returns:
+        str: snapshot hash
+    """
     return create_snapshot(path, message)
 
 
+#TODO: docstring: `contains` parameter
+#TODO: verify accuracy of docstring
 def list_snapshots(bucket, contains=None):
```

Comment:
So, basically, I made a TODO because this param is strangely named for its functionality -- "If `contains` starts with the snapshot's path, include the snapshot in the results."

..is this the intended functionality?